editor/components/TableMap.tsx

```diff
@@ -88,42 +97,49 @@ export const TableMap = ({ mapRef, geojson, mapConfig }: TableMapProps) => {
   }, []);
 
   return (
-    <MapGL
-      ref={mapRef}
-      initialViewState={initialViewState}
-      {...DEFAULT_MAP_PARAMS}
-      cooperativeGestures={true}
-      onMouseEnter={onMouseEnter}
-      onMouseLeave={onMouseLeave}
-      cursor={cursor}
-      onLoad={(e) => e.target.resize()}
-      maxZoom={21}
-      interactiveLayerIds={["points-layer"]} // layer id defined in mapConfig
-      onClick={(e) => {
-        e.originalEvent.stopPropagation();
-        if (e.features?.length) {
-          setSelectedFeature(e.features[0]);
-        } else {
-          setSelectedFeature(null);
-        }
-      }}
-    >
-      <FullscreenControl position="bottom-right" />
-      <NavigationControl position="top-right" showCompass={false} />
-      {/* custom geojson source and layer */}
-      <Source id="custom-source" type="geojson" data={geojson}>
-        <Layer type="circle" {...mapConfig?.layerProps} />
-      </Source>
-      <MapFitBoundsControl mapRef={mapRef} bounds={geojsonBounds} />
-      {selectedFeature && (
-        <PopupWrapper
-          longitude={selectedFeature?.properties?.longitude}
-          latitude={selectedFeature?.properties?.latitude}
-          featureProperties={selectedFeature.properties}
-          PopupContent={TableMapPopupContent}
-          onClose={() => setSelectedFeature(null)}
+    !!basemapType && (
```

Comment:
this value is not nullable, so maybe we don't need this conditional render?

Reply:
this is correct, remnants from a previous iteration where it was nullable :smile: ty!